.github/workflows/deploy-staging.yml

```diff
@@ -26,6 +26,7 @@ jobs:
             if [ ! -d socialpredict-deploy ]
             then
                 git clone ${{ github.server_url }}/${{ github.repository }} socialpredict-deploy
+                cd socialpredict-deploy
```

Comment:
I guess I missed this during one of the last pulls, but isn't it potentially risky to use `cd` in a yaml / shell script due to some of the problems associated with relative paths?

When I first was running socialpredict as a series of scripts, I had a script written that would cd in and out of the various directories to build the dockerfiles, it was a mess. I'm not sure if that applies here, different application, just a passing thought.